### PR TITLE
Feature/improved debug logging

### PR DESCRIPTION
**Pull request by hedvigoscar**

### CODE REVIEW

File: src/utils/log.ts [hunk 1]
```diff
@@ -4,24 +4,25 @@ import { SegmentAnalyticsJs } from '../interfaces'
...
+    console.groupEnd()
     analytics.track(...args)
   }
   const identify: SegmentAnalyticsJs['identify'] = (...args) => {
-    console.log('%c[IDENTIFY]', blueBold, {
-      userId: args[0],
-      traits: args[1],
-      options: args[2],
-    })
+    console.group('%c[IDENTIFY]', blueBold)
```
Comment (palmfjord) [inline src/utils/log.ts]: this is really cool

Comment (palmfjord) [reply src/utils/log.ts]: although... is this turned off in prod?

Comment (palmfjord) [reply src/utils/log.ts]: i feel like it should be

Comment (hedvigoscar) [reply src/utils/log.ts]: It's called conditionally if `QuepasaOptions.debug == true` - totally opt-in. In the web-onboarding branch I'm working in right now I've set it to be conditional on `process.env.NODE_ENV === 'development`

Comment (palmfjord) [reply src/utils/log.ts]: ooh amazeballs!

**Pull request merged**